app/controllers/api/vcs.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+use Utopia\App;
+use Appwrite\Event\Build;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Helpers\Permission;
+use Utopia\Database\Helpers\Role;
+use Appwrite\Utopia\Request;
+use Appwrite\Utopia\Response;
+use Utopia\Validator\Text;
+use Utopia\VCS\Adapter\Git\GitHub;
+use Utopia\Database\Helpers\ID;
+use Appwrite\Extend\Exception;
+use Utopia\Cache\Adapter\Redis;
+use Utopia\Cache\Cache;
+use Utopia\Database\Query;
+use Utopia\Database\Adapter\MariaDB;
+use Utopia\Database\Validator\Authorization;
+
+App::get('/v1/vcs/github/install')
```

Comment:
Our REST API is designed to have CRUD operations for nouns, `install` is a verb which is for example more common for an API like GraphQL or for one of our SDKs. What would work better for consistency with other Appwrite APIs is to use a noun like `installations`.

For our SDK we would just use `createInstallation` for consistency and following a pattern that have proved to work without conflicts.